librz/core/cmd_descs/cmd_search.yaml

```diff
@@ -190,19 +190,8 @@ commands:
             optional: true
             type: RZ_CMD_ARG_TYPE_STRING
   - name: "/c"
-    summary: Cryptographic material search.
+    summary: Cryptographic material & hash search.
```

Comment:
```suggestion
    summary: Cryptographic material search.
```

Since it wasn't implemented here yet.

Reply:
yes but soon it will, and probably it will changed again for other constants.